docs/cli/descheduler.md

```diff
@@ -19,10 +19,10 @@ descheduler [flags]
       --client-connection-kubeconfig string      File path to kube configuration for interacting with kubernetes apiserver.
       --client-connection-qps float32            QPS to use for interacting with kubernetes apiserver.
       --descheduling-interval duration           Time interval between two consecutive descheduler executions. Setting this value instructs the descheduler to run in a continuous loop at the interval specified.
-      --disable-http2-serving                    If true, HTTP2 serving will be disabled [default=false]
+      --disable-http2-serving                    If true, HTTP2 serving will be disabled [default=false] (default true)
```

Comment:
don't love this part :/ `[default=false] (default true)`

Reply:
Oh, `[default=false]` is part of the usage string. The descheduler code could access the usage string and drop `[default=false]`. I.e. calling `VisitAll` and editing `Flag.Usage` field. Worth trying that.